pyproject.toml

```
@@ -32,7 +32,7 @@ dependencies = [
     "nomad-lab>=1.3.4dev", 
     "xmltodict==0.13.0",
     "fairmat-readers-xrd~=0.0.3",
-    "pynxtools", 
+    "pynxtools>=0.6.0",
```

```suggestion
    "pynxtools>=0.6.1",
```

Let's update the pynxtools to the version where you made the changes in `populate_nexus_subsection`